src/components/BugsTable/BugsTableHead.jsx

```diff
@@ -0,0 +1,78 @@
+import { Component } from 'react';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import TableSortLabel from '@material-ui/core/TableSortLabel';
+import Tooltip from '@material-ui/core/Tooltip';
+import PropTypes from 'prop-types';
```

Comment:
Instead of importing all of prop-types, only import the specific types you are going to use.